src/main/io/osd.c

```diff
@@ -1346,6 +1350,14 @@ static void osdUpdateStats(void)
         }
     }
 #endif
+    value = ((tempInDeciDegrees * 9) / 5) + 320;
```

Comment:
You should not restrict this to only fahrenheit units.  Instead use the `osdConfig()->units` to determine whether to calculate celsius or fahrenheit.  It's probably better to just determine the max here in native units and do the final conversion when the stat is displayed.